ipa-core/src/net/client/mod.rs

```diff
@@ -180,6 +180,7 @@ impl MpcHelperClient {
         peer_config: PeerConfig,
         identity: ClientIdentity,
     ) -> Self {
+        setup_crypto_provider();
```

Comment:
I don't think this is the right place to setup crypto provider. The defaults must be setup at the application layer (their [doc](https://docs.rs/rustls/latest/rustls/crypto/struct.CryptoProvider.html#using-a-specific-cryptoprovider) suggests that too).

Maybe a good alternative would be to call https://docs.rs/rustls/latest/rustls/client/struct.ClientConfig.html#method.builder_with_provider instead.